tuxemon/client.py

```diff
@@ -184,6 +184,26 @@ def load_map(self, map_data: TuxemonMap) -> None:
         self.event_engine.reset()
         self.event_engine.current_map = map_data
         self.maps = map_data.maps
+        self.map_name = map_data.name
+        self.map_desc = map_data.description
+
+        # Cardinal points
+        if map_data.north_trans == None:
```

Comment:
This is a bit fussy, but when comparing to None you should do `is None` not `== None`.

They both work, but `is None` is considered the proper way.